html/index.html

```diff
@@ -534,6 +531,22 @@ <h1>Sampling and Sequencing Simulator</h1>
   return adjusted_values;
 }
 
+function inverse_transform_sample(cdf) {
```

Comment:
This looks correct.

While thinking about the problem I came across this [faster algorithm](https://en.wikipedia.org/wiki/Alias_method) for generating discrete random variables. No reason to switch to it unless this somehow becomes the bottleneck step.